ibis/expr/api.py

```diff
@@ -648,6 +648,35 @@ def random() -> ir.FloatingScalar:
 
     Similar to [](`random.random`) in the Python standard library.
 
+    ::: {.callout-note}
+    ## Repeated use of `random`
+
+    `ibis.random()` will generate a column of distinct random numbers even if
+    the same instance of `ibis.random()` is re-used.
+
+    When Ibis compiles an expression to SQL, each place where `random` is used
+    will render as a separate call to the given backend's random number
+    generator.
+
+    ```{python}
+    >>> from ibis.interactive import *
+    >>> t = ibis.memtable({"a": range(5)})
+    >>> r_a = ibis.random()
+    >>> t.mutate(random_1=r_a, random_2=r_a)  # doctest: +SKIP
+    ┏━━━━━━━┳━━━━━━━━━━┳━━━━━━━━━━┓
```

Comment:
I think I originally suggested using `{python}` but perhaps that wasn't a good idea 😅 , since the entire block is parsed as Python and so these unicode blocks are causing syntax errors.

Can you try removing the `{}` and see if that works?

Sorry for the churn!

Reply:
Well, looks like you read my mind!

Reply:
yeah, weird interaction between quarto and doctest